integrations/lib/testing/integration/integration_test.go

```diff
@@ -40,7 +40,7 @@ func (s *IntegrationSuite) TestVersion() {
 
 	versionMin, err := version.NewVersion("v11.0.0")
 	require.NoError(t, err)
-	versionMax, err := version.NewVersion("v13")
+	versionMax, err := version.NewVersion("v14")
```

Comment:
Do we have to update this with each major version?

Reply:
Yes, otherwise tests will fail
Maybe we can get the correct version from `/version.go` and then increment by one
Increment by one because that's actually our compatibility guidelines, right? Clients must support Auth/Proxy with vX+1 (where X is the client's version)